.github/workflows/CI.yml

```diff
@@ -0,0 +1,72 @@
+name: CI
+
+on: [ push, pull_request, workflow_dispatch ]
+
+jobs:
+  build_zip:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ghidra: [ 'latest', '10.1.3', '10.1.2', '10.1.1' ]
+    steps:
+      - name: Set Ghidra version
+        id: ghidra_version
+        run: |
+          if [ ${{ matrix.ghidra }} = latest ]; then
+            ghidra_version=$(curl -fLsSo /dev/null -w %{url_effective} https://github.com/NationalSecurityAgency/ghidra/releases/latest |
+             grep -oP '(?<=https://github.com/NationalSecurityAgency/ghidra/releases/tag/Ghidra_).+(?=_build)')
+            echo "::set-output name=full::$ghidra_version"
+          else
+            echo "::set-output name=full::${{ matrix.ghidra }}"
+          fi
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Checkout pspdev/psp-ghidra-scripts
+        uses: actions/checkout@v3
+        with:
+          repository: 'pspdev/psp-ghidra-scripts'
+          path: 'psp-ghidra-scripts'
+      - name: Setup Java
+        uses: actions/setup-java@v3
+        with:
+          java-version: 11
+          distribution: temurin
+      - name: Setup Ghidra
+        uses: er28-0652/setup-ghidra@master
+        with:
+          version: ${{ steps.ghidra_version.outputs.full }}
+      - name: Build Extension
+        run: |
+          ls ${{ env.GHIDRA_INSTALL_DIR }}
+          chmod +x gradlew
+          ./gradlew shadowJar
+      - name: Prepare artifact
+        run: |
+          mkdir -p Allegrex/lib
+          cp build/libs/ghidra-allegrex-all.jar Allegrex/lib/Allegrex.jar
+          cp -R data ghidra_scripts Allegrex
+          cp -R psp-ghidra-scripts/* Allegrex/ghidra_scripts
```

Comment:
I don't think we can pack those, that repository doesn't provide any license information.

The build step should just use `ghidraInstallThenPackage`, we need to include compiled Sleigh too.

Reply:
Using `ghidraInstallThenPackage` fails for some reason, I listed `GHIDRA_INSTALL_DIR` at the start https://github.com/illusion0001/ghidra-allegrex/runs/6991790602?check_suite_focus=true#step:7:46

Reply:
> I don't think we can pack those, that repository doesn't provide any license information.

Removed.

Reply:
Also tried some other stuff yesterday, those failed too. https://github.com/illusion0001/ghidra-allegrex/runs/6971463398?check_suite_focus=true#step:6:46 the edited build file: https://github.com/illusion0001/ghidra-allegrex/blob/0744e5a940c92d94ca79b99ab3f1bd2c87700a8b/build.gradle#L139

Reply:
I think the command is incorrect, admittedly I only tested it on Windows. Looking at Ghidra files it should be just `sleigh` without `sh`. (btw Gradle should have some `--debug` or `--info` flag to print more details)

Reply:
I own the `psp-ghidra-scripts` repo and can add license info

Reply:
@John-K thanks but do you think it's worth bundling those two together? I'm not sure.
For now I'll add a link to `psp-ghidra-scripts` in the README.

@illusion0001 I removed the `.sh` suffixes from build file so it will probably work now.

Reply:
Yea, build works now https://github.com/illusion0001/ghidra-allegrex/runs/7161891756?check_suite_focus=true#step:6:43